csireverseproxy/pkg/standaloneproxy/standaloneproxy.go

```diff
@@ -382,6 +382,7 @@ func (revProxy *StandAloneProxy) GetRouter() http.Handler {
 	router.PathPrefix(utils.Prefix + "/{version}/system/symmetrix/{symid}").HandlerFunc(revProxy.ServeReverseProxy)
 	router.PathPrefix(utils.Prefix + "/{version}/sloprovisioning/symmetrix/{symid}").HandlerFunc(revProxy.ServeReverseProxy)
 	router.PathPrefix(utils.PrivatePrefix + "/{version}/replication/symmetrix/{symid}").HandlerFunc(revProxy.ServeReverseProxy)
+	router.PathPrefix(utils.PrivatePrefix + "/{version}/sloprovisioning/symmetrix/{symid}").HandlerFunc(revProxy.ServeReverseProxy)
```

Comment:
There was a missing private handler for sloprovisioning and our request was dropped

Reply:
This seems to indicate that all requests using that API will get dropped. Was this the original problem?